api/tests/routes/public/individual_offers/v1/post_product_test.py

```diff
@@ -620,3 +623,61 @@ def test_returns_404_for_inactive_venue_provider(self, client):
         )
 
         assert response.status_code == 404
+
+
+def build_base_payload(venue, **extra):
+    return {
+        "location": {"type": "physical", "venueId": venue.id},
+        "categoryRelatedFields": {
+            "category": "SUPPORT_PHYSIQUE_FILM",
+            "ean": "1234567891234",
+        },
+        "accessibility": utils.ACCESSIBILITY_FIELDS,
+        "name": "Le champ des possibles",
+        **extra,
+    }
+
+
+@pytest.fixture(name="venue_offerer")
+def venue_offerer_fixture():
+    return utils.create_offerer_provider_linked_to_venue()
+
+
+@pytest.fixture(name="venue")
+def venue_fixture(venue_offerer):
+    return venue_offerer[0]
+
+
+@pytest.fixture(name="offerer")
+def offerer_fixture(venue_offerer):
+    return venue_offerer[1]
+
+
+@pytest.fixture(name="auth_client")
+def auth_client_fixture(client, venue):
+    return client.with_explicit_token(offerers_factories.DEFAULT_CLEAR_API_KEY)
+
+
+@pytest.mark.usefixtures("db_session")
+class CreateProductDbErrorsHandlingTest:
+    endpoint = "public_api.post_product_offer"
```

Comment:
Il vaut mieux utiliser directement l'url pour deux raisons : 
- ça permet de catcher un breaking change si, par mégarde, on change l'url du endpoint (ce qui est toujours bon a catcher pour une API publique qui doit être stable)
- c'est plus facile à lire pour le développeur (`/public/offers/v1/products` on sait tout de suite de quoi on parle `public_api.post_product_offer` un peu moins)

```suggestion
    product_url = "/public/offers/v1/products"
```

Reply:
Ok pourquoi pas, je modifie. Mais rapidement :  

1. si c'est juste pour détecter un changement d'URL, on peut écrire un test dédié (on n'est pas obligé de faire planter tous les tests pour ça 😅 ).
2. c'est subjectif 😺  depuis un éditeur, c'est simple d'accéder à la fonction du contrôleur avec son nom (et c'est appréciable de pouvoir y accéder rapidement). Tandis que c'est plus compliqué de la retrouver à partir d'une URL. Et pour savoir de quoi on parle, le nom du fichier donne aussi du contexte. C'est une question de préférence au final.